runbotics/runbotics-actions-windows/src/automation/office/excel.types.ts

```diff
@@ -63,4 +64,10 @@ export interface CellCoordinates {
     endRow?: number;
 }
 
-export type ExcelArrayStructure = unknown[][]
+export type ExcelInsertColumnsInput = {
+    column: string;
+    amount?: number;
```

Comment:
it's 1 for default? we shoud inform about this in tooltip. Check if everything goes fine when amount of columns not specified

Reply:
if you haven't done it yet ofc :D

Reply:
previously it was, but now the amount input mandatory as well (it was suggested to me that since the action is named insert columns it will be more clear for the user to make this decision)